lib/install/management/appliance.go

```diff
@@ -118,8 +118,61 @@ func (d *Dispatcher) isContainerVM(vm *vm.VirtualMachine) (bool, error) {
 
 func (d *Dispatcher) checkExistence(conf *config.VirtualContainerHostConfigSpec, settings *data.InstallerData) error {
 	defer trace.End(trace.Begin(""))
-
 	var err error
+
```

Comment:
why was this needed?

Reply:
This code was added because we create folders at the vmfolder level. So even though the vch may be unique in a cluster it may not be unique in the inventory. We needed to check in the inventory to make sure that the vch pathing would not collide. I have added and unblocked some tests in this PR that were broken because of us not checking for existence before hand. It originally was not going to be needed when we namespaced with the compute. but now that we make the vch folder directly on the vm folder we have to have uniqueness enforced at that level as well.